### PR TITLE
Add some fixes for red text removal

### PR DESCRIPTION
**Pull request by alayek**
Added some fixes for red text removal. This is to be merged with the new branch `workshopper` that @koustuvsinha created.

@koustuvsinha @abhisekp please check.

### CODE REVIEW

File: provision.sh [hunk 1]
```diff
@@ -41,14 +41,15 @@ echo "-------- Installing NVM & Node --------------"
...
+su - vagrant -c "wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.31.1/install.sh | bash >/dev/null 2>&1"
 
 # source bash and install node
 su - vagrant <<NVM
 
 source ~/.nvm/nvm.sh
-echo 'sourcing done'
-nvm install 4.4.3
+echo 'Sourcing done'
+nvm install 4.4.3 > /dev/null 2>&1
```
Comment (abhisekp) [inline provision.sh:51]: It think it is OK to present the error if is not installed due to some reason. 
Moreover in the next line, even if nvm command is unsuccessful, we'll get a fake successful message.

**Comment by abhisekp** [comment]
IMHO, error should appear at least in these initial versions.

File: provision.sh (hunk 1 above)
Comment (alayek) [inline provision.sh:51]: No, it pipes the output to `/dev/null`, but any error would come up in output because of `2>&1`..

Comment (alayek) [inline provision.sh:51]: Also, these echo statements are mainly for us, developers of vagrant boxes, as checkpoints for debugging.

**Comment by alayek** [comment]
@abhisekp I agree. But here we are only addressing end-users' concern of not wanting to see some red text here and there in midst of green text - vagrant by default pipes anything headed for `STDERR` to red color text. Even if it's npm install warnings.

We are not hiding the errors - we are simply making their color same as `STDOUT` green.

**Comment by koustuvsinha** [comment]
# LGTM :+1:

will merge it after #2 closes

**Comment by abhisekp** [comment]
@koustuvsinha This need to be merged to #2 and then #2 can be merged

**Pull request merged**